BMM.Core/Implementations/Analytics/AppCenterAnalytics.cs

```diff
@@ -30,27 +31,21 @@ public void LogEvent(string eventName, IDictionary<string, object> parameters)
             var user = _userStorage.GetUser();
             if (user != null)
             {
-                if (_remoteConfig.UseAnalyticsId)
-                    AddIfNew(parameters, nameof(user.AnalyticsId), user.AnalyticsId);
-                else
-                    AddIfNew(parameters, nameof(user.PersonId), user.PersonId);
+                parameters.AddIfNew(nameof(user.AnalyticsId), user.AnalyticsId);
+                parameters.AddIfNew(nameof(user.PersonId), user.PersonId);
```

Comment:
don't log the personId (see above comment). We use the anonymized AnalyticsId instead.